packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/src/components/footer.module.css

```diff
@@ -24,27 +20,31 @@
 	margin: var(--x-auto);
```

Comment:
since the value of `--x-auto` is `0 auto`, it will not get converted, tailwind would understand plain `auto`